MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -5,6 +5,8 @@
 import UIKit
 
 public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching {
+	@IBOutlet private(set) var errorView: ErrorView!
```

Comment:
Following the same comment in MVP module `errorView` getter can be public

```suggestion
	@IBOutlet public private(set) var errorView: ErrorView!
```